modin/pandas/series.py

```diff
@@ -1948,15 +1948,21 @@ def to_numpy(
         """
         Return the NumPy ndarray representing the values in this Series or Index.
         """
-        return (
-            super(Series, self)
-            .to_numpy(
-                dtype=dtype,
-                copy=copy,
-                na_value=na_value,
+        from modin.config import ExperimentalNumPyAPI
```

Comment:
import should be done on top of the file

Reply:
Also, I wonder, if NumPy API is experimental feature, why do we put it as a core one?

Reply:
Actually, I mean if NumPy API is experimental, we should probably put it under modin/experimental/numpy, shouldn't we.

Reply:
@RehanSD, @vnlitvinov, ping

Reply:
IMO, we want folks to be able to access the NumPy API as a top level module, so it makes sense here. The experimental part is directly converting df's to modin.numpy instead of actual numpy, but I'm open to other folks' opinions. Pinging @devin-petersohn

Reply:
I'd prefer to keep `modin.numpy` instead of `modin.experimental.numpy`. We already throw a warning when the module is imported, so that should be enough, and I don't want users to have to change their code as they upgrade to a more stable "non-experimental" version.

Reply:
> I don't want users to have to change their code as they upgrade to a more stable "non-experimental" version.

Sure, that makes sense to keep `modin.numpy` instead of `modin.experimental.numpy` right away. However, this is kind of unfair to other experimental APIs we have. We could move those to the layer up then.